src/main/java/br/edu/ufcg/computacao/alumni/core/holders/AlumniHolder.java

```diff
@@ -160,6 +161,13 @@ private synchronized String getToken(String eurecaBackendPublicKey) throws Eurec
         return alumniMap;
     }
 
+    private String parseResponseContent(HttpResponse response) {
+        String content = response.getContent();
```

Comment:
Add a comment here. This code is rather obscure.

Reply:
Done